docs/architecture/klickerKnight.md

```diff
@@ -0,0 +1,20 @@
+# Klicker Knight
+
+[klickerKnight.ts](../src/klickerKnight.ts) represents the entry point to the game.
```

Comment:
This should be `../../src/klickerKnight.ts`

Reply:
Updated this one and all of the ones I could find. We can find or add a linting rule for any other ones